{
  "comments": [
    {
      "key": {
        "uuid": "df87a7cf_7882f7ca",
        "filename": "doc/source/configuration.rst",
        "patchSetId": 3
      },
      "lineNbr": 248,
      "author": {
        "id": 16068
      },
      "writtenOn": "2017-12-02T10:40:34Z",
      "side": 1,
      "message": "nit: when you\u0027re at it: Defaults",
      "range": {
        "startLine": 248,
        "startChar": 4,
        "endLine": 248,
        "endChar": 11
      },
      "revId": "b17516eeff3f1cf44e468ef7fcf2893d9059a015",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "df87a7cf_38554f51",
        "filename": "nodepool/tests/__init__.py",
        "patchSetId": 3
      },
      "lineNbr": 297,
      "author": {
        "id": 16068
      },
      "writtenOn": "2017-12-02T10:40:34Z",
      "side": 1,
      "message": "not introduced by you, but the indentation looks wrong (which is undetected by the currently broken flake8 rules in nodepool)",
      "range": {
        "startLine": 295,
        "startChar": 0,
        "endLine": 297,
        "endChar": 78
      },
      "revId": "b17516eeff3f1cf44e468ef7fcf2893d9059a015",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "df87a7cf_18e16b9b",
        "filename": "nodepool/tests/test_drivers.py",
        "patchSetId": 3
      },
      "lineNbr": 31,
      "author": {
        "id": 16068
      },
      "writtenOn": "2017-12-02T10:40:34Z",
      "side": 1,
      "message": "maybe you additionally could use an external copy of the fake driver and not just load but really use the driver",
      "revId": "b17516eeff3f1cf44e468ef7fcf2893d9059a015",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    }
  ]
}